addons/account/models/account_journal_dashboard.py

```diff
@@ -79,13 +79,14 @@ def _query_has_sequence_holes(self):
             SELECT move.journal_id,
                    move.sequence_prefix
               FROM account_move move
-             WHERE move.journal_id = ANY(%(journal_ids)s)
+              JOIN account_journal journal ON journal.id = move.journal_id
```

Comment:
I would filter in `_compute_has_sequence_holes` instead.
It can still make sense to run this query on other journals at other places.